scalafix-core/shared/src/main/scala/scalafix/internal/config/ScalafixConfig.scala

```diff
@@ -77,7 +77,7 @@ object ScalafixConfig {
     // Are extractor varargs specified using colons, i.e. is `case Extractor(xs: _*)` legal or not?
     allowColonForExtractorVarargs = true,
     // Are inline vals and defs supported by this dialect?
-    allowInlineMods = true,
+    allowInlineMods = false,
```

Comment:
what's this about?

Reply:
I'm trying to minimize the breaking change by switching the parser. If a rule previously used `inline` as an identifier then it would compile with reflect before this commit but not after. It's the only custom flag in DefaultDialect that removes features from the original dialect, the other flags only enable the dialect to parse more files.

Reply:
Clear, thanks!